src/Employer/Employer.Web/Extensions/ApplicationReviewExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore.Projections.VacancyApplications;
+
+namespace Esfa.Recruit.Employer.Web.Extensions
+{
+    public static class ApplicationReviewExtensions
+    {
+        public static string GetFriendlyId(this ApplicationReview applicationReview)
```

Comment:
May be make this an extension to guid (ToShortId)

Reply:
I thought about that but didn't want to encourage its use elsewhere.